shell/platform/windows/public/flutter_windows.h

```diff
@@ -203,6 +203,27 @@ FLUTTER_EXPORT void FlutterDesktopEngineSetNextFrameCallback(
     VoidCallback callback,
     void* user_data);
 
+typedef int64_t PlatformViewId;
```

Comment:
Can all these be moved to the internal header file too?

Reply:
Only if we include the internal header file from the implementation files that use these typedefs. I don't know if this is desirable or not.

Reply:
As in is it desirable if things like the platform view manager include the internal header file That seems acceptable to me, what concerns do you have with that?

Reply:
Simply that we will need to remember to remove those includes if/when all the necessary declarations are moved into the public header file.

Reply:
Ah gotcha, that's fair. It's very likely we have unnecessary `#include`s all over the place already.

That said, the benefits of having a "pub-ternal" APIs that we can experiment with and evolve is well worth some the risk of unnecessary `#include`s in the future.